src/renderers/webgl/WebGLBackground.js

```diff
@@ -15,7 +15,7 @@ import { ShaderLib } from '../shaders/ShaderLib.js';
 function WebGLBackground( renderer, state, geometries, premultipliedAlpha ) {
 
 	var clearColor = new Color( 0x000000 );
-	var clearAlpha = 0;
+	var clearAlpha = 1;
```

Comment:
Changing the default `clearAlpha`to 1 will cause a lot of breakage. I think `clearAlpha` should remain at 0.